### PR TITLE
Add Results Prometheus histogram config

### PR DESCRIPTION
**Pull request by enarha**
# Changes

This adds the PROMETHEUS_HISTOGRAM configuration option existing in Tekton Results.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Run `make test lint` before submitting a PR
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/operator/v1alpha1/tektonresult_types.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ type ResultsAPIProperties struct {
 	DBEnableAutoMigration *bool  `json:"db_enable_auto_migration,omitempty"`
 	ServerPort            *int64 `json:"server_port,omitempty"`
 	PrometheusPort        *int64 `json:"prometheus_port,omitempty"`
+	PrometheusHistogram   *bool  `json:"prometheus_histogram,omitempty"`
```
Comment (jkandasa) [inline pkg/apis/operator/v1alpha1/tektonresult_types.go:74]: @enarha thanks for the PR.
IMHO: the naming will be easy readable, if something like
```
EnablePrometheusHistogram   *bool  `json:"enable_prometheus_histogram,omitempty"`
```

I believe the naming from result component.

**Review by jkandasa — APPROVED** [review]

**Comment by jkandasa** [comment]
@enarha can you please include documentation for this change, if possible

File: pkg/apis/operator/v1alpha1/tektonresult_types.go (hunk 1 above)
Comment (enarha) [reply pkg/apis/operator/v1alpha1/tektonresult_types.go:74]: The name in Results is `PROMETHEUS_HISTOGRAM` https://github.com/tektoncd/results/blob/main/cmd/api/README.md?plain=1#L23, so I just used that name. I thought this is the convention we follow, at least it is for the other configuration options.

**Pull request merged**